### PR TITLE
Convert task runner to jobloop

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

File: internal/tasks/convert.go [hunk 1]
```diff
@@ -45,67 +46,38 @@ const (
...
+func (c *Context) processConversion(_ context.Context, tx *gorp.Transaction, pd tenso.PendingDelivery, labels prometheus.Labels) (returnedError error) {
 	var event tenso.Event
 
 	defer func() {
-		labels := prometheus.Labels{"source_payload_type": event.PayloadType, "target_payload_type": pd.PayloadType}
+		labels["source_payload_type"] = event.PayloadType
+		labels["target_payload_type"] = pd.PayloadType
 		if event.PayloadType == "" { //because we did not get to loading it
 			labels["source_payload_type"] = "early-db-access"
 		}
```
Comment (majewsky) [inline internal/tasks/convert.go:77]: This is now prefilled by the library. As a general remark that also applies to the other job, it is now recommended to fill labels within the regular flow of the ProcessRow function, as soon as the label values become available. (If processing fails before that point, the `early-db-access` default value is retained.)

**Review by majewsky — CHANGES_REQUESTED** [review]
The `Context.Blocker` mechanism and associated unit tests can be removed. Parallelism tests are covered in go-bits/jobloop now.

File: internal/tasks/convert.go (hunk 1 above)
Comment (SuperSandro2000) [reply internal/tasks/convert.go:77]: Can/Should we assert that somehow by the library?

Comment (SuperSandro2000) [reply internal/tasks/convert.go]: Also shuffled the label filling to the earliest safe position.

**Review by majewsky — APPROVED** [review]
This is ready to merge once you're ready to adjust the alert definitions and dashboards.

**Comment by SuperSandro2000** [comment]
> This is ready to merge once you're ready to adjust the alert definitions and dashboards.

I couldn't find any 😆

**Pull request merged**